docs/UserGuide.md

```diff
@@ -345,8 +340,11 @@ Parameters Restrictions:
 
 * The optional index refers to the index number shown in the displayed person list. The index **must be a positive
   integer** 1, 2, 3, …​
+* If the optional index is not provided, all loans, across all clients in the list will be shown.
```

Comment:
Should we say all active loans here?

Reply:
I didn't include it because the next line immediately after this talks about the active loans.

Reply:
Hmm that is true, but I feel like this could indicate a problematic behaviour in how `all loans` are distinguished from `all loans, including inactive ones` in the UG. This is because when we say `all loans`, it is _implicitly_ implied that they do not include inactive ones, but a user (especially one who has not read anything on `all loans, including inactive ones`) might misinterpret that to mean the latter.

Regardless, I'll approve these changes first and we could address this issue in a separate PR.